python/graphstorm/gconstruct/remap_result.py

```diff
@@ -786,9 +888,17 @@ def generate_parser():
                        default=None,
                        help="The directory storing the node embeddings.")
     group.add_argument("--output-format", type=str,
-                       default="parquet",
-                       choices=['parquet'],
+                       default=GS_OUTPUT_FORMAT_PARQUET,
+                       choices=[GS_OUTPUT_FORMAT_PARQUET, GS_OUTPUT_FORMAT_CSV],
                        help="The format of the output.")
+    group.add_argument("--output-delimiter", type=str, default=",",
+                       help="The delimiter used when saving data in CSV format.")
+    group.add_argument("--column-names", type=str, nargs="+", default=None,
+                       help="Defines how to rename default column names to new names."
+                       f"For example, given --column-names {GS_REMAP_NID_COL}:~id "
+                       f"{GS_REMAP_EMBED_COL}:embedding. The column "
```

Comment:
will it be clear what GS_REMAP_NID_COL and GS_REMAP_EMBED_COL variable names mean?

Reply:
We will create build a doc to introduce the output of save embeddings.